charts/timed/values.yaml

```
@@ -307,13 +305,19 @@ redmine:
   url: redmine.example.com
   # redmine.apiKey -- Redmine API Key
   apiKey: ""
-  # redmine.htaccessUser -- Redmine htaccess user
-  htaccessUser: ""
-  # redmine.htaccessPassword -- Redmine htaccess password
-  htaccessPassword: ""
   # redmine.spenthoursField -- Spent hours field id on Redmine
   spenthoursField: 0
 
+sentry:
+  # sentry.enabled -- Enable Sentry integration
+  enabled: false
+  # sentry.dsn -- Sentry DSN for error reporting. Set to enable Sentry integration
+  dsn: sentry.example.com
```

using `nil` forces the user to overwrite this:
```suggestion
  dsn: ~
```